charts/images.yaml

```diff
@@ -88,7 +93,7 @@ images:
   sourceRepository: github.com/gardener/autoscaler
   repository: eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler
   tag: "v1.20.3"
-  targetVersion: "< 1.21"
+  targetVersion: "1.20.x" #gardener supports K8S version >=1.20 only. See https://github.com/gardener/gardener/blob/master/docs/usage/supported_k8s_versions.md#:~:text=Gardener%20itself%20is%20capable%20of%20spinning%20up%20clusters%20with%20Kubernetes%20versions%201.20%20up%20to%201.25 for reference.
```

Comment:
Is there a special reason for this change? There might be landscapes which run shoots with older versions than Kubernetes v1.20 and they would not have a `cluster-autoscaler` anymore. This would be the first component being so picky in the entire file.

Reply:
Yes I agree , we would remove the change.

Reply:
removed the change